### PR TITLE
5.0 wip

### PR DESCRIPTION
**Pull request by mbpframework**
MySqlConnector 0.69.9 --->MySqlConnector 1.0.1

### CODE REVIEW

**Comment by mguinness** [comment]
This PR is redundant, please see 5.0-wip branch where MySqlConnector was updated to version 1.0.1 and also [Nightly Builds](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#nightly-builds).

**Comment by lauxjpn** [comment]
@mbpframework What is the purpose of this PR? I only see commits by myself of the past that have already been merged.

> MySqlConnector 0.69.9 --->MySqlConnector 1.0.1

> This PR is redundant, please see 5.0-wip branch where MySqlConnector was updated to version 1.0.1 and also [Nightly Builds](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#nightly-builds).

Also see #1198, #1161, #1153, #1130, #1126 and #1103.

**Comment by mbpframework** [comment]
the lateset package on the nuget.org is only support MySqlConnector 0.69.9

**Comment by lauxjpn** [comment]
> the lateset package on the nuget.org is only support MySqlConnector 0.69.9

From #1198:

> This is a duplicate of #1161.
> Everything of interest to you should be in there.
> 
> TLDR: The prereleases of Pomelo 5.0 already supports MySqlConnector >= 1.0.0. There will not be added support in Pomelo < 5.0.

---

Also, take a look at the issues I referenced above:

> Also see #1198, #1161, #1153, #1130, #1126 and #1103.

From #1161:

> This is expected. MySqlConnector changed its namespace in its 1.0.0 version, which is a breaking change. Since we strife to be [SemVer2](https://semver.org/) compliant, we cannot introduce breaking changes in minor or patch version releases, but only in major version releases.
> 
> Therefore, current Pomelo releases have been published with the constraint, that a MySqlConnector version below 1.0.0 must be used. From the Dependencies section on NuGet:
> 
> > MySqlConnector (>= 0.69.9 && < 1.0.0)
> 
> This is also stated in [Compatibility: Dependencies](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#dependencies) and the description of [release 3.2.0](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/releases/tag/3.2.0) (and the previous one):
> 
> > This release is compatible with .NET Standard 2.0, EF Core 3.1.8, MySQL 8.0/5.7, MariaDB 10.5/10.4/10.3 and **MySqlConnector 0.69.9 (but <1.0.0)**. [...]
> 
> Also related is PR #1103:
> 
> > Since this will be a breaking change, we should make this PR part of the `5.0.0` branch, and let `3.x.x` continue using the last available `MySql.Data` namespace release.
> 
> So Pomelo's `5.0+` releases will be using the latest MySqlConnector versions again.

---

As a side note, your PR targets the `5.0-wip` branch, only contains commits that have already been merged and not a single commit made by you (at least according to GitHub's commit list above). It doesn't matter now, because our current 5.0 builds already support MySqlConnector 1.0.0+, so this PR is not going to be merged anyway. But I though I mention it anyway.
